src/agent.ts

```diff
@@ -35,6 +37,17 @@ export const getKeepAliveAgentOptions = (env: typeof process.env) => ({
       : parseInt(env.FETCH_KEEPALIVE_TIMEOUT, 10)
 });
 
+export const getKeepAliveHttpsAgentOptions = (
+  env: typeof process.env
+): agentkeepalive.HttpsOptions => ({
+  ...getKeepAliveAgentOptions(env),
+  cert: env.FETCH_TLS_CERT,
```

Comment:
since we need different certs for different client, we'll need to override these params so the usage will be:

`getKeepAliveHttpsAgentOptions({...process.env, FETCH_TLS_CERT: tlsCert, passPhrase: FETCH_TLS_PASSPHRASE, ...})`

The passphrase should be optional anyway (ie. INPS certs have no passphrase AFAIK @francescopersico)

Reply:
uhm at this point it doesn't make much sense to have them parsed by `getKeepAliveHttpsAgentOptions`, the cert params can be passed directly to `newHttpsAgent`, e.g:

``ts
const config = getKeepAliveHttpsAgentOptions(process.env);

const agent1 = newHttpsAgent({
  ...config,
  cert: process.env.INPS_TLS_CERT,
  key: process.env.INPS_TLS_KEY
})
```